packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputValue.tsx

```diff
@@ -67,6 +69,7 @@ export function GridFilterInputValue(props: GridTypeFilterInputValueProps & Text
   React.useEffect(() => {
     return () => {
       clearTimeout(filterTimeout.current);
+      setIsApplying(false);
```

Comment:
```suggestion
```

Reply:
This needs to be there because there is no more local state in the `GridFilterForm`. Before the `return` was only called once, now its every time the value changes.